### ISSUE TITLE
Light Table's Future.

### ISSUE THREAD

**Issue by prertik** [LightTable/LightTable#2484]
https://lighttable.discourse.group/t/future-of-lighttable/50/2?u=prertik

**Comment by Mouvedia** [comment]
For the ones wondering what's CIDER's Orchard: https://metaredux.com/posts/2018/11/09/ciders-orchard-the-heart.html

**Comment by allentiak** [comment]
@prertik I just checked your presentation. 


> But, the new codebase will be open-sourced, once, I've fixed Analyzing plugins.

Are you sure about this? I think this will only difficult receiving feedback... Why don't just use a different branch?

**Comment by prertik** [comment]
@allentiak Yes, I'm sure. In fact, this is the only viable approach to make it one true Clojure editor.
Yeah, I'll use a different branch, I was thinking maybe of another repo or something like that. I want to keep this codebase for legacy reasons.

**Comment by allentiak** [comment]
BTW, @prertik: Chances are, "Please check this out" is probably not the best subject choice...
Have you tried something like "Light Table's Future", or the like?

**Comment by prertik** [comment]
Thanks for the suggestion @allentiak! BTW, did you vote?

**Comment by allentiak** [comment]
Vote? What for?

**Comment by allentiak** [comment]
Vote? On what? I don't understand the question...

**Comment by sbauer322** [comment]
I believe he may be referring to this tweet, @allentiak - https://twitter.com/lighttable/status/1233018553827115008?s=20

**Comment by allentiak** [comment]
*Disclaimer: I share my opinion in the hope that it might be useful,
and because @prertik asked for it. I will be clear: even though I
fancy the idea of reviving LT, actually working towards it is not on
my priority list right now. Of course, this might change in the
future, but it is too soon to tell.


@prertik Nice new title!

Thanks, @sbauer322! I knew nothing about this poll.

I think a new release is critical - even if that code is thrown away
right away. After all, all code eventually ends up in the trash bin :-)

And I think that the sooner a new release is out, the better. That
could be useful for reviving the project. Once it is revived, the new
LT community members may decide where to take it from there. But be
willing to spend a few months working all alone. Maybe recruiting a
local (this is: physically close) colleague/mate might help?

Regarding Community Management, I can advise the following readings:

- [How to Have a Successful Free Software
Project](https://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.87.4806)
"[T]here is more to the free software development than the creation of
a ‘bazaar’." Great place to start. Awesome reading. In fact, I think I
should re-read it myself :-)

 - [Open Advise](http://www.open-advice.org/), a knowledge collection
from a wide variety of Free Software projects. It answers the question
what 42 prominent contributors would have liked to know when they
started so you can get a head-start no matter how and where you
contribute.

- [The Cathedral and the
Bazaar](http://www.catb.org/~esr/writings/cathedral-bazaar/) Eric
Raymond's journey to becoming (and quitting being) Sendmail's
maintainer.

- [Producing OSS](https://producingoss.com/) This is more a reference book.

I hope this helps. Happy hacking!

**Comment by niklas-heer** [comment]
Are there any updates already? 
I would really like to see this project being revived. 👍
